econet_heatpump_water_heater.yaml

```diff
@@ -108,10 +108,21 @@ text_sensor:
   - platform: econet
     name: "Model Number"
     sensor_datapoint: PRODMODN
+    request_mod: 2
+    request_once: true
     icon: "mdi:information-box"
     entity_category: "diagnostic"
   - platform: econet
     name: "Serial Number"
     sensor_datapoint: PRODSERN
+    request_mod: 2
+    request_once: true
+    icon: "mdi:information-box"
+    entity_category: "diagnostic"
+  - platform: econet
+    name: "Software Version Number"
+    sensor_datapoint: SW_VERSN
+    request_mod: 2
+    request_once: true
```

Comment:
I'm not aware of any firmware update mechanism but are we sure this can't change?

Reply:
Even if it can change via firmware update, no harm not updating it. At the next ESPHome reset it will get the right value.

Reply:
Moved to base